src/screens/FeedScreen/ContentLink.js

```diff
@@ -98,11 +99,11 @@ const ContentLink = ({
 export default ContentLink;
 
 const styles = StyleSheet.create({
-  contentFeed: {
-    flex: 1,
+  contentFeed: (hasComment) => ({
+    flex: hasComment ? 1 : 'none',
     marginHorizontal: 6,
     backgroundColor: COLORS.white
-  },
+  }),
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `flex` property of the `contentFeed` style is now conditionally set based on the `hasComment` prop. This is a good way to adjust the layout based on the presence of comments. However, using 'none' for flex might cause issues as 'none' is not a valid value for flex in React Native. It should be 0 if you want to disable flex.
```diff
-    flex: hasComment ? 1 : 'none',
+    flex: hasComment ? 1 : 0,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->